src/SZ/config.h

```diff
@@ -0,0 +1,4 @@
+/* Static config.h file added for hdf5plugin to spare running the configuration */
+#if !defined(_WIN32) || defined(__MINGW32__)
+#define HAVE_UNISTD_H
+#endif
```

Comment:
I was wondering whether to put it in `src/SZ/` or in a separate folder e.g., `src/SZ_extra/`

Reply:
I guess in a separate folder is better in order not to mix our code with theirs.

Reply:
OK, done!